src/PPN-microbench/CMakeLists.txt

```
@@ -12,13 +10,18 @@ target_sources(
         BASE_DIRS ${PROJECT_SOURCE_DIR}/include
         FILES ${PROJECT_SOURCE_DIR}/include/PPN-microbench/microbench.hpp
         FILES ${PROJECT_SOURCE_DIR}/include/PPN-microbench/cpu_frequency.hpp
+        FILES ${PROJECT_SOURCE_DIR}/include/PPN-microbench/context.hpp
+        FILES ${PROJECT_SOURCE_DIR}/include/PPN-microbench/conductor.hpp
         FILES ${PROJECT_SOURCE_DIR}/include/PPN-microbench/ops/abstract_ops.hpp
         FILES ${PROJECT_SOURCE_DIR}/include/PPN-microbench/ops/flops.hpp
         FILES ${PROJECT_SOURCE_DIR}/include/PPN-microbench/ops/iops.hpp
```

Files can be passed together without re-using `FILES` (also prefer relative file paths):
```suggestion
        FILES
            microbench.hpp
            cpu_frequency.hpp
            context.hpp
            conductor.hpp
            ops/abstract_ops.hpp
            ops/flops.hpp
            ops/iops.hpp
```